releasenotes/notes/replace-tags-metrics-31c05b88d3f356f8.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    APM: DD_APM_REPLACE_TAGS aka apm_config.replace_tags will now properly look for tags with numeric values.
```

Comment:
```suggestion
    APM: The ``DD_APM_REPLACE_TAGS`` environment variable and ``apm_config.replace_tags`` setting now properly look for tags with numeric values.
```

Reply:
Just a note that this is the cherry-pick PR, so we will also need to apply this to the other PRs.